rubocop-sequel.gemspec

```diff
@@ -4,7 +4,8 @@ Gem::Specification.new do |gem|
   gem.authors       = ['Timothée Peignier']
   gem.email         = ['timothee.peignier@tryphon.org']
   gem.description   = 'Code style checking for Sequel'
-  gem.summary       = 'A Sequel plugin for the RuboCop code style & linting tool.'
+  gem.summary       = 'A Sequel plugin for the RuboCop code style & ' \
```

Comment:
Or we can just shorten the description to "A Sequel plugin for RuboCop".

Reply:
I updated it. Thanks!